docs/v1.0/sinks/datadog.md

```diff
@@ -32,4 +32,5 @@ sinks:
     metric-translator: ""
     metric-prefix: ""
     dogstatsd-host: ""
-```
+    api-use-compression: "true"
```

Comment:
`api-compression`  (or `api-compress` if you want to mimic the DD client). Blip doesn't use words like "enable", "use", etc. The only such on/off type word is the consistent prefix `disable-...`.